resources/styles/book-content/theming-mixins.less

```diff
@@ -16,13 +16,19 @@
 .tutor-book-theme-note(@label-background: @tutor-book-primary; @label-text: @tutor-book-text-on-dark-labels) {
   > .note,
   > section > .note,
-  section > section > .note {
+  section > section > .note, .grasp-check {
```

Comment:
can you put each selector on a new line? That way the line does not, at a glance, look like `section > section > .note .grasp-check`